src/setup.js

```diff
@@ -27,9 +50,62 @@ function setupMeasure(dataLayer) {
    * The DataLayerHelper to use with this application.
    * @private @const {!DataLayerHelper}
    */
-  const helper = new DataLayerHelper(dataLayer);
+  const helper = new DataLayerHelper(dataLayer, {'processNow': false});
+
+  helper.registerProcessor('config', configProcessors);
+  helper.process();
+
+  /**
+   * When called with an implementation of the eventProcessor and
+   * storageInterface API, this function registers processors to react to the
+   * 'set' and 'event' commands.
+   *
+   * @param {!ProcessorConstructor|string} eventProcessor The event
+   *     processor to use when responding to event commands
+   *     and deciding how keys should be stored, or a string
+   *     representing an object of this type.
+   * @param {!Object<string, *>} eventOptions Options objects to pass to the
+   *    event processor.
+   * @param {!StorageConstructor|string} storageInterface An
+   *     interface to a class that will handle setting
+   *     and loading long term data, or a string representing
+   *     an object of this type.
+   * @param {!Object<string, *>} storageOptions Options objects to pass to the
+   *    storage interface.
+   */
+  function configProcessors(eventProcessor, eventOptions,
```

Comment:
Prefer arrow functions for nested functions: https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions

However, I'd lean towards defining configProcessors in a separate module and just using it here. I don't know what the tagging team wants to do, but there may be a desire for some pubs to compile this library directly into their code. Having a separate module for each of these may make adapting this library easier for that use case.

Reply:
I moved it to a new file